Frends.ServiceBus.Read/Frends.ServiceBus.Read/Read.cs

```diff
@@ -0,0 +1,271 @@
+﻿using System;
+using System.ComponentModel;
+using System.Threading;
+using System.Threading.Tasks;
+using Frends.ServiceBus.Read.Definitions;
+using Microsoft.Azure.ServiceBus;
+using Microsoft.Azure.ServiceBus.Core;
+using System.Text;
+using Microsoft.Azure.ServiceBus.Management;
+using Microsoft.Azure.ServiceBus.InteropExtensions;
+using System.Net.Mime;
+using System.Linq;
+
+namespace Frends.ServiceBus.Read;
+
+/// <summary>
+/// Azure Service Bus task.
+/// </summary>
+public class ServiceBus
+{
+    /// <summary>
+    /// Read message from Azure Service Bus queue or topic.
+    /// </summary>
+    /// <param name="input">Input parameters</param>
+    /// <param name="options">Options parameters</param>
+    /// <param name="cancellationToken">Token generated by Frends to stop this task.</param>
+    /// <returns>Object { bool ReceivedMessage, string Content, string ContentType, string SessionId, string MessageId, string CorrelationId, int DeliveryCount, long EnqueuedSequenceNumber, long SequenceNumber, string Label, Dictionary&lt;string, object&gt; Properties }, string ReplyTo, string ReplyToSessionId, long Size, string To, DateTime ScheduledEnqueueTimeUtc</returns>
+    public static async Task<Result> Read([PropertyTab] Input input, [PropertyTab] Options options, CancellationToken cancellationToken)
```

Comment:
You can make multi-line comments. No need to keep these all in the same line.

Reply:
I think it's cleaner to keep these in 1 line because you can word wrap these lines in Visual Studio when using smaller resolution.